src/data/helpText.yaml

```diff
@@ -1,29 +1,28 @@
 product:
-  name: ""
+  name: "Name of offering (ie. your product or service)."
   version: "Provide version if available."
-  description: ""
+  description: "Describe the general offering. This should not be specific to a contract or RFP."
 author:
-  name: ""
-  company_name: ""
-  address: ""
-  email: ""
-  phone: ""
-  website: ""
+  name: "Contact name or team representing author of the ACR (optional)."
+  company_name: "Name of the accessibility agency that wrote the ACR (optional)."
+  email: "Include a contact email for questions or feedback on the ACR."
+  phone: "Accessibility agency phone number (optional)."
+  website: "Website for accessibility agency (optional)."
 vendor:
-  name: ""
-  company_name: ""
-  address: ""
-  email: ""
-  phone: ""
-  website: ""
+  name: "Contact name or team representing the vendor (optional)."
+  company_name: "Name of the vendor who is responsible for the offering or is submitting the the ACR."
+  address: "Street address for vendor (optional)."
+  email: "Include a contact email for questions or feedback on the ACR (optional)."
+  phone: "Vendor phone number (optional)."
+  website: "Website for the vendor or offering (optional)."
 report:
-  report_date: ""
-  notes: ""
-  evaluation_methods_used: ""
-  legal_disclaimer: ""
-  repository: ""
-  feedback: ""
-  license: ""
+  report_date: "Date last modified (YYYY/MM/DD)."
```

Comment:
The current is actually 'MM/DD/YYYY'. Should I change the format to match the help text?

Reply:
I was just making one up, but that's obviously been done too many times..  I think we should settle on some standard.  
https://iso.mit.edu/americanisms/date-format-in-the-united-states/

I've changed it to DD/MM/YYYY (and we should be consistent).  I could also be convinced by an ISO argument https://www.iso.org/iso-8601-date-and-time-format.html

But would be good not to be country specific.